### ISSUE TITLE
[U1/UT99->UT3/4] Wrong Texture Alignment when PanU/V set

### ISSUE THREAD

**Issue by xtremexp** [xtremexp/UT4X-Converter#2]
Polygons with Pan U/V  values have bad texture alignement for Unreal 1/Unreal Tournament maps
It can be really annoying to manually fix all textures unaligned afterwards.

DmFith converted to UT3 with unaligned texture :

![Image](https://user-images.githubusercontent.com/11919600/208874192-36e0b68e-1915-432c-af3b-00484ffd2a86.png)

Original t3d polygon surface data:

```
Begin Polygon Item=OUTSIDE Texture=Starship.Panel.s4 Link=2
	Origin   +00000.000000,-01216.000000,+00000.000000
	Normal   +00000.000000,+00000.000000,+00001.000000
	TextureU +00000.500000,+00000.000000,+00000.000000
	TextureV +00000.000000,+00000.500000,+00000.000000
	Pan      U=160 V=64
	Vertex   -00064.000000,-00064.000000,+00000.000000
	Vertex   +00064.000000,-00064.000000,+00000.000000
	Vertex   +00064.000000,+00064.000000,+00000.000000
	Vertex   -00064.000000,+00064.000000,+00000.000000
End Polygon
```

Converted to UT3 (with 1.25X scale):
```
 Begin Polygon Item=Side Texture=DmFith.Starship_Panel_s4_Mat Flags=3584 Link=2 ShadowMapScale=64.000000
    Origin   +01520.000000,+00000.000000,+00000.000000
    Normal   +00000.000000,+00000.000000,+00001.000000
    TextureU +00000.000000,+00000.800000,+00000.000000
    TextureV -00000.800000,+00000.000000,+00000.000000
    Vertex   +00080.000000,-00080.000000,+00000.000000
    Vertex   +00080.000000,+00080.000000,+00000.000000
    Vertex   -00080.000000,+00080.000000,+00000.000000
    Vertex   -00080.000000,-00080.000000,+00000.000000
  End Polygon
```
Different tests to see how it works.
TextureU and TextureV are used for texture scaling and rotation.
Origin, PanU, PanV are used for texture position.

PanU=0, Origin=(0,0,0)
![image](https://user-images.githubusercontent.com/11919600/210130903-862a9a48-1c39-42d2-9dab-b7eb29b2cab3.png)

PanU=16, Origin=(0,0,0)
![image](https://user-images.githubusercontent.com/11919600/210130599-e5ffc1ec-9af5-438c-99ff-17fe00a284f5.png)

PanU=0, Origin=(-16,0,0)
![image](https://user-images.githubusercontent.com/11919600/210130606-0821ae8b-43e8-4825-9cd3-8b6e4941a6e5.png)

**Comment by xtremexp** [comment]
Update: partially fixed in 1.2.1 still some remaining textures with bad alignment.

**Comment by xtremexp** [comment]
Finally this painful is fixed, all textures are now correctly aligned.
DM-ArcaneTemple (Original):
![image](https://user-images.githubusercontent.com/11919600/212470246-ccd53f48-dea7-456c-952d-3a29e2355839.png)

DM-ArcaneTemple (Converted):
![image](https://user-images.githubusercontent.com/11919600/212470258-e94cf5e4-312a-417d-a8b1-563789750c44.png)
